crates/core/tedge/src/cli/certificate/create_csr.rs

```diff
@@ -0,0 +1,165 @@
+use super::create::cn_of_self_signed_certificate;
+use super::error::CertError;
+use crate::command::Command;
+use crate::CreateCertCmd;
+use camino::Utf8PathBuf;
+use certificate::NewCertificateConfig;
+
+pub struct CreateCsrCmd {
+    pub id: Option<String>,
+    pub cert_path: Utf8PathBuf,
+    pub key_path: Utf8PathBuf,
+    pub csr_path: Utf8PathBuf,
+    pub output_path: Option<Utf8PathBuf>,
```

Comment:
We should have only one parameter here as both have the same role.

```suggestion
    pub csr_path: Utf8PathBuf,
```